src/nbody/plotter.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.animation import FuncAnimation
+
+def plotter(filename):
+
+    # Extracting the positions of each body at each time step
+    df = pd.read_csv(filename)
+    bodies = df['bodies'].unique()
+    time = df['time'].unique()
+    data = {}
+    for body in bodies:
+        new_df = df[df['bodies'] == body][['px', 'py']]
+        data[body] = new_df.to_numpy()     
+
```

Comment:
we could also add something like this, since the objects we read in from our simulation will have other attributes, that we can later use to inform attributes of the animation (e.g. coloring objects in certain mass ranges differently or something)

```suggestion
has_mass = 'mass' in df.columns
    has_radius = 'radius' in df.columns
    has_v = {'vx', 'vy'}.issubset(df.columns)

    body_objs = []
    for body in bodies:
        mask = df['bodies'] == body
        mass = df.loc[mask, 'mass'].iloc[0] if has_mass else None
        radius = df.loc[mask, 'radius'].iloc[0] if has_radius else None
        v_arr = df.loc[mask, ['vx', 'vy']].to_numpy() if has_v else None

        body_objs.append(
            Body(
                id=str(body),
                mass=mass,
                radius=radius,
                pos_arr=data[body],  # reuse your existing data dict
                v_arr=v_arr
            )
        )
```